.github/workflows/ci.yml

```diff
@@ -0,0 +1,552 @@
+        env:
+          MARIADB_VERSION: 10.6
+      - name: "Run tests"
+        run: mvn -B test -pl repository -am -Dtest=AllDBTestsTestSuite -DfailIfNoTests=false -Ddb.name=alfresco -Ddb.url=jdbc:mariadb://localhost:3307/alfresco?useUnicode=yes\&characterEncoding=UTF-8 -Ddb.username=alfresco -Ddb.password=alfresco -Ddb.driver=org.mariadb.jdbc.Driver
+      - name: "Clean Maven cache"
+        if: ${{ always() }}
+        run: bash ./scripts/ci/cleanup_cache.sh
+
+  repository_mysql_tests:
+    name: Repository - MySQL 8 tests
+    runs-on: ubuntu-latest
+    needs: [prepare]
+    if: >
+      (((github.ref_name == 'master' || contains(github.ref_name, 'release/') || github.event_name == 'pull_request') &&
+      ! contains(github.event.head_commit.message, '[skip db]')) ||
+      contains(github.event.head_commit.message, '[latest db]') ||
+      contains(github.event.head_commit.message, '[db]')) &&
+      ! contains(github.event.head_commit.message, '[skip tests]') &&
+      ! contains(github.event.head_commit.message, '[force]')
+    steps:
+      - uses: actions/checkout@v3
+      - uses: Alfresco/acs-community-packaging/.github/actions/get-build-info@feature/ACS-3844_migrate-to-GHA
+      - uses: Alfresco/alfresco-build-tools/.github/actions/setup-java-build@v1.25.0
+      - name: "Init"
+        run: bash ./scripts/ci/init.sh
+      - name: "Set up the environment"
+        run: docker-compose -f ./scripts/ci/docker-compose/docker-compose-db.yaml --profile mysql up -d
+        env:
+          MYSQL_VERSION: 8
+      - name: "Run tests"
+        run: mvn -B test -pl repository -am -Dtest=AllDBTestsTestSuite -DfailIfNoTests=false -Ddb.driver=com.mysql.jdbc.Driver -Ddb.name=alfresco -Ddb.url=jdbc:mysql://localhost:3307/alfresco -Ddb.username=alfresco -Ddb.password=alfresco
+      - name: "Clean Maven cache"
+        if: ${{ always() }}
+        run: bash ./scripts/ci/cleanup_cache.sh
+
+  repository_postgresql_13_7_tests:
+    name: "Repository - PostgreSQL 13.7 tests"
+    runs-on: ubuntu-latest
+    needs: [prepare]
+    if: >
+      (((github.ref_name == 'master' || contains(github.ref_name, 'release/')) && github.event_name != 'pull_request' &&
+      ! contains(github.event.head_commit.message, '[skip db]')) ||
+      contains(github.event.head_commit.message, '[db]')) &&
+      ! contains(github.event.head_commit.message, '[skip tests]') &&
+      ! contains(github.event.head_commit.message, '[force]')
+    steps:
+      - uses: actions/checkout@v3
+      - uses: Alfresco/acs-community-packaging/.github/actions/get-build-info@feature/ACS-3844_migrate-to-GHA
+      - uses: Alfresco/alfresco-build-tools/.github/actions/setup-java-build@v1.25.0
+      - name: "Init"
+        run: bash ./scripts/ci/init.sh
+      - name: "Set up the environment"
+        run: docker-compose -f ./scripts/ci/docker-compose/docker-compose-db.yaml --profile postgres up -d
+        env:
+          POSTGRES_VERSION: 10.6
```

Comment:
this doesn't look consistent with the name of the job

Reply:
Good catch. I fixed it. By the way, I renamed some "Set up the environment" steps to "Run [database name] database" if only the database container is started.